prisma/migrations/20250203223348_add_residents/migration.sql

```diff
@@ -0,0 +1,14 @@
+-- CreateTable
+CREATE TABLE "Resident" (
+    "id" TEXT NOT NULL,
+    "tenantId" TEXT,
+    "name" TEXT NOT NULL,
+    "phoneNumber" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    CONSTRAINT "Resident_pkey" PRIMARY KEY ("id")
+);
+
+-- AddForeignKey
+ALTER TABLE "Resident" ADD CONSTRAINT "Resident_tenantId_fkey" FOREIGN KEY ("tenantId") REFERENCES "Tenant"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
hmm why does it do an onDelete set null

Reply:
I believe this is normal
https://github.com/prisma/prisma/issues/16228